packages/go_router_builder/README.md

```diff
@@ -93,71 +101,83 @@ class HomeRoute extends GoRouteData {
 
 The tree of routes is defined as an attribute on each of the top-level routes:
 
+<?code-excerpt "example/lib/main.dart (RouteTree)"?>
 ```dart
 @TypedGoRoute<HomeRoute>(
   path: '/',
   routes: <TypedGoRoute<GoRouteData>>[
     TypedGoRoute<FamilyRoute>(
-      path: 'family/:familyId',
-    )
+      path: 'family/:fid',
+// ···
```

Comment:
Let's include the closing `)` here, even though it'll mean an extra plaster. Same for similar cases later in the file; it's just less confusing if we don't have unbalanced `([{`s.